.circleci/config.yml

```diff
@@ -177,7 +177,9 @@ jobs:
             - v2-back-dependencies-{{ .Revision }}
       - run:
           name: Install gettext (required to make messages)
-          command: sudo apt-get install -y gettext
+          command: |
+            sudo apt-get update
```

Comment:
Is it worth doing this because it's very long? Maybe not if the `python:3.10` image is always updated?

Reply:
With this image, we had to fetch the latest list of available packages elsewhere it is not able to install gettext.

https://app.circleci.com/pipelines/github/openfun/richie/3596/workflows/c8dee5f9-ef49-4269-af45-b74dd3655974/jobs/128485

Reply:
ok!